prod-config.yml

```diff
@@ -77,6 +77,20 @@ metadata:
     path: prod_ida_metadata.ts
     password: puppet
 
+# Including this section will enable European identities and the universal dataset for matching
+# European identities are NOT enabled for production yet so this should remain false
+europeanIdentity:
+  enabled: false
+  hubConnectorEntityId: https://www.signin.service.gov.uk/SAML2/metadata/connector
+
+  # Details about eIDAS metadata can be configured here
+  aggregatedMetadata:
+    trustAnchorUri: https://www.signin.service.gov.uk/SAML2/metadata/trust-anchor
+    metadataSourceUri: https://www.signin.service.gov.uk/SAML2/metadata/aggregator
+    trustStore:
+      path: prod_ida_metadata.ts
+      password: puppet
+
```

Comment:
As per discussed, please try this "condensed down" version of the config to make sure that it works.